### PR TITLE
feature/managed-coh

### PR DESCRIPTION
**Pull request by chpatel3**
Provide an option shown below to the domain input file for enabling coherence on managed servers.

# Option to enable coherence on managed servers
# The following line must be uncommented and updated with desired cluster name to enable coherence.
#coherenceClusterName: Cluster-0

### CODE REVIEW

**Comment by chpatel3** [comment]
@rjeberhard Please review this PR to develop branch.

**Comment by markxnelson** [comment]
@chpatel3 please review the requirements for contributing here: https://github.com/oracle/weblogic-kubernetes-operator/blob/master/CONTRIBUTING.md 
You will need to be a signatory of the OCA before we can merge any PR from you.

**Comment by markxnelson** [comment]
Or if you are an Oracle employee, you need to be a part of the Oracle GitHub Organization.  Ping me if you need a link to the process.

File: kubernetes/create-weblogic-domain-inputs.yaml [hunk 1]
```diff
@@ -96,3 +96,7 @@ loadBalancerDashboardPort: 30315
 
 #Java Option for Weblogic Server
 javaOptions: -Dweblogic.StdoutDebugEnabled=false
+
+# Option to enable coherence on managed servers
```
Comment (rjeberhard) [inline kubernetes/create-weblogic-domain-inputs.yaml]: Upper case "Coherence"

File: kubernetes/create-weblogic-domain-inputs.yaml [hunk 2]
```diff
@@ -96,3 +96,7 @@ loadBalancerDashboardPort: 30315
...
+# The following line must be uncommented and updated with desired cluster name to enable coherence.
+#coherenceClusterName: Cluster-0
```
Comment (rjeberhard) [inline kubernetes/create-weblogic-domain-inputs.yaml]: Add validation that coherenceClusterName specifies an cluster in the domain.

**Comment by rjeberhard** [comment]
@moreaut Can you take a look?

@chpatel3 I'm okay with including this work in the near term.  We are working on a rewrite to the domain home generation that would use the WebLogic Deploy Tooling model.

**Pull request closed without merging**